code/game/gamemodes/changeling/powers/transform.dm

```diff
@@ -7,17 +7,138 @@
 	req_human = 1
 	max_genetic_damage = 3
 
+//changeling clothing items
+/obj/item/clothing/head/changeling

```

Comment:
I'm fairly certain most of the inventory slots are only defined as /item, meaning any coded assuming they're NOT /item, are already in the wrong.